solutions/cses-1706.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: cses-1706
+source: CSES
+title: School Excursion
+author: Óscar Garries
+---
+## Solution
+**Time complexity:**$\mathcal{O}(N\log^2N)$
+
+The solution of the problem is based on DSU ( [Disjoin Set Union](https://usaco.guide/gold/dsu?lang=cpp) )
+For example: if 2 and 3 want  the same thing and 3 and 4 want the same think ==> It means that 2, 3 and 4 want the same thing. After we make the components of the DSU, we are going to use their sizes. After that, we need to find out every combination of these sizes. To do that, we gonna use a knapsack DP ( you can find a good tutorial [here](https://usaco.guide/gold/knapsack?lang=cpp)). But it would tale too long to do this, it would take O(N ^ 2) => TLE. In order to speed it up, we gonna use a bitset as a knpasack.
+
+
+```cpp
+#include <bitset>
```

Comment:
Use tabs for indentation

Reply:
they are used; the autoformatter enforces this

Reply:
The code looks ok, do you want to use tabs of 4 spaces ?